src/scss/constants.scss

```diff
@@ -116,6 +116,8 @@ $font-weight--400: 400;
 $font-weight--500: 500;
 $font-weight--600: 600;
 $font-weight--700: 700;
+$font-weight--800: 800;
+$font-weight--900: 900;
```

Comment:
Suitable replacements for hardcoded `font-size: 800` and `font-size 900` found in legacy code, now with the capability to quickly find usages through IDE.